app/packs/src/apps/mydb/elements/details/cellLines/propertiesTab/CellLineName.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import CellLinesFetcher from 'src/fetchers/CellLinesFetcher';
+import PropTypes from 'prop-types';
+import { StoreContext } from 'src/stores/mobx/RootStore';
+import {
+  Col, Row, ControlLabel, FormControl
+} from 'react-bootstrap';
+import Creatable from 'react-select3/creatable';
+
+export default class CellLineName extends React.Component {
+  // eslint-disable-next-line react/static-property-placement
+  static contextType = StoreContext;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      nameSuggestions: []
+    };
+  }
+
+  componentDidMount() {
+    CellLinesFetcher.getAllCellLineNames()
+      .then((data) => data.map((x) => ({ value: x.id, label: x.name, name: x.name })))
```

Comment:
There can be duplicates because the key is {name, source}. in a next iteration i will add the source also to the visible attributes in the select menu.